app/views/shared/_user_info_dropdown.html.erb

```diff
@@ -3,11 +3,11 @@
     <!-- User Info Dropdown Menu -->
     <div id="login-info" class="login-info-dropdown">
```

Comment:
would the class you are adding work for the individual items if you added it to this parent `div`, instead of adding it to each of the items?

Reply:
that sounds right, but I think it has to be tried. looks like these are 'a' tags, they should inherit from a parent div, but maybe they have styles that override?

Reply:
I tried that and it look likes none of the characters wrap if the class is in the `div`. So ` Change Provider User's Guide Logout` are all on a single line.